src/main/java/com/javadiscord/javabot/events/InteractionListener.java

```diff
@@ -26,10 +28,11 @@ public void onButtonClick(ButtonClickEvent event) {
 		Guild guild = preferredGuild;
 		MongoDatabase database = mongoClient.getDatabase("other");
 		String[] id = event.getComponentId().split(":");
+		System.out.println(event.getComponentId());
```

Comment:
It is cleaner if you annotate the class with `@Slf4j` and use `log.info(event.getComponentId())` instead.

Reply:
well, i actually forgot to remove that